CHANGELOG.md

```diff
@@ -5,13 +5,15 @@
 Changelog since 1.4.1:
 ### Added
  - Add `Modern690Lc.downgrade_to_legacy` (unstable API)
+ - Added a bash completion script for tab completion
 ### Changed
  - Accept hexadecimal inputs regardless of a `0x` prefix
  - Warn on faulty temperature readings from Kraken X3 coolers
  - Warn on Hydro Platinum/PRO XT firmware versions that are may be too old
  - Update PyInstaller used for the Windows executable
  - Update PyUSB bundled with the Windows executable
  - Improve the documentation
+ - Updated installation instructions for the manpage
```

Comment:
Not necessary, the previous item already covers this.

Reply:
didnt notice that, Ill remove this in the next revision